core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5167,7 +5167,20 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
-
+  public static final PropertyKey TABLE_UDB_HIVE_CLIENTPOOL_MIN =
+      new Builder(Name.TABLE_UDB_HIVE_CLIENTPOOL_MIN)
+          .setDefaultValue("16")
+          .setDescription("The minimum capacity of the hive client pool per hive metastore")
```

Comment:
If these pools are per UDB, then should they be UDB properties instead? (meaning, each `attach`ed db can have a different sized pool.)

Reply:
took another closer look, these pools are per connection, not per UDB. since we don't have properties associated with connectionUris, i am inclined to keep them as global property keys for now. we can address this in a separate PR if needed.